apps/backend/src/authn/oidc.strategy.ts

```diff
@@ -36,14 +37,26 @@ export class OidcStrategy extends PassportStrategy(Strategy, 'oidc') {
         userInfoURL: configService.get('OIDC_USER_INFO_URL') || 'disabled',
         clientID: configService.get('OIDC_CLIENTID') || 'disabled',
         clientSecret: configService.get('OIDC_CLIENT_SECRET') || 'disabled',
-        callbackURL: `${configService.get('EXTERNAL_URL')}/authn/oidc/callback`,
-        scope: 'openid profile email'
+        callbackURL: `${configService.get('EXTERNAL_URL')}/authn/oidc_callback`,
+        pkce:
+          configService.get('OIDC_USES_PKCE_S256') === 'true'
+            ? 'S256'
+            : configService.get('OIDC_USES_PKCE_PLAIN') === 'true'
+              ? 'plain'
+              : undefined,
```

Comment:
Are we catching that this could be undefined anywhere else?

Reply:
the type of the pkce attribute is `pkce?: 'S256' | 'plain'`.  setting it explicitly to undefined makes it so that we are disabling pkce support.